blt/blt.yml

```diff
@@ -231,6 +231,11 @@ validate:
       - files.php.tests
 
 tests:
+  cypress:
+    basePath: ${repo.root}/tests/cypress
```

Comment:
This should also control the number of threads to use. If you look at https://github.com/NCIOCPL/cgov-digital-platform/blob/d7671e9d0ff82fe16aebe6b0c6eea2c3b6109904/blt/blt.yml#L241, you will see that ci.blt.yml sets the threads to 3. This is because we found that locally 2 was the best, but Github actions could handle 3.

Reply:
Done